### PR TITLE
feat(query): added Azure Active Directory Authentication for Terraform

### PR DESCRIPTION
**Pull request by tiagocruz20**
**Proposed Changes**
- added a query to check if Azure Active Directory is defined for authentication for Service Fabric

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/azure/azure_active_directory_authentication/query.rego [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package Cx
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/azure_active_directory_authentication/query.rego:1]: We could include a rule to check if `azure_active_directory` is undefined

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/azure/azure_active_directory_authentication/query.rego [hunk 2]
```diff
@@ -15,3 +15,17 @@ CxPolicy[result] {
 		"keyActualValue": sprintf("'aazurerm_service_fabric_cluster[%s].azure_active_directory.tenant_id' is undefined or null", [name]),
 	}
 }
+
+CxPolicy[result] {
+	azure := input.document[i].resource.azurerm_service_fabric_cluster[name]
+
+	not common_lib.valid_key(azure, "azure_active_directory")
+
+	result := {
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/azure_active_directory_authentication/query.rego:24]: Missing searchLine

**Review by rafaela-soares — CHANGES_REQUESTED** [review]

File: assets/queries/terraform/azure/azure_active_directory_authentication/query.rego [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package Cx
+
+import data.generic.common as common_lib
+
+CxPolicy[result] {
+	active := input.document[i].resource.azurerm_service_fabric_cluster[name].azure_active_directory
+
+	not common_lib.valid_key(active, "tenant_id")
+
+	result := {
```
Comment (rafaela-soares) [inline assets/queries/terraform/azure/azure_active_directory_authentication/query.rego:10]: Missing searchLine

**Review by rafaela-soares — DISMISSED** [review]

**Review by felipe-avelar — APPROVED** [review]
LGTM

**Review by rogeriopeixotocx — APPROVED** [review]
LGTM

**Pull request merged**